rust/index/src/utils.rs

```diff
@@ -77,6 +79,15 @@ pub fn merge_sorted_vecs_conjunction<T: Ord + Clone>(a: &[T], b: &[T]) -> Vec<T>
     result
 }
 
+pub fn normalize(vector: &[f32]) -> Vec<f32> {
```

Comment:
nit - this same function exists elsewhere this is copy pasted. Also @Sicheng-Pan was moving it to a place it was easy to export. Can we benefit from that or mark this TODO?